### ISSUE TITLE
Schematic Submission 

### ISSUE THREAD

**Issue by timChase98** [mbisbano1/ECE388_CatchTheLightV2#2]
- [x] While this is very organized its hard to follow.  
  - [x] Try to have clear flow from left-right 
  - [x] Power input top left
  - [x] Buttons input to uController
  - [x] uController outputs to LEDs 
- [x] Do you have reverse voltage protection for if the user inserts a battery backwards? 
- [x] Use power flags instead of named nets for power
- [x] Any time a signal leaves the section of the schematic that should be clearly indicated by a tail net. LED_ANODE is an example of this.  It should be clear that that is coming from somewhere else
- [x] The 328PB symbol doesn't need to be anywhere near that wide.  
- [x] Decoupling capacitors should be located near the chip they are decoupling.  Ideally connected via a physical net, this makes it more clear what chip it belongs to when you go to do a layout.  

Aside from format issue this looks really good.  Address the format issues but I will move onto board review.  

Please address these issues and push your changes. Then create a new release, increment the tag and close this issue. I will review the changes and mark this as complete or reopen the issue with additional feedback.

**Comment by mbisbano1** [comment]
I have tried to clear all issues posed here. 

1. Added Schottky Diode Reverse voltage protection
2. Replaced net labels with Power Flags from supply1 library
3. Made Signals entering and exiting nets easier to follow by using bold label flags.
4. Narrowed the 328PB symbol.. 
5. Placed Decoupling capacitors physically closer to 328pb symbol to denote their proximity on board layout.
6. More organized left - right flow of information in schematic.

- Battery connection top left.
- Immediately below that is the reverse voltage protection and power switch
- Immediately below that is the current limiting resistor which 'produces' LED_ANODE voltage source.
- Flow of Button/Spare Button signal connection is more intuitive as well, which is directly connected to uController PE3 through a physical net.

NOTE:
- I left the LEDs along the top row and did not directly connect them to the 328pb symbol as the clutter from connecting all of those traces alongside with the decoupling caps would surely make the schematic unreadable.

**Comment by timChase98** [comment]
Why are there resistors on the programming line? Its also called a bed of nails programmer not a plug of nails.  Fix those things and I'll give you a pass

**Comment by mbisbano1** [comment]
Issues fixed as requested.

**Comment by timChase98** [comment]
Change your reverse voltage protection to the one from [THIS](https://www.ti.com/lit/an/slva139/slva139.pdf) App note from TI.  That should be the last thing.

**Comment by mbisbano1** [comment]
@timChase98 I copied the Schottky diode reverse voltage protection from that TI App note.. Are you suggesting the MOSFET protection instead?

**Comment by timChase98** [comment]
Yes the point of the app note is that just dropping a diode in isn't a great solution and using the NFet is much better

**Comment by mbisbano1** [comment]
Schottky Diode reverse protection replaced with a PMOS protection circuit citing said app note. Reasoning for PMOS is PMOS are cheaper than NMOS by thin margin, and wiring is easier in my scenario.
